index.yaml

```diff
@@ -94,9 +94,87 @@ entries:
     - gateway-2.0.1.tgz
     version: 2.0.1
   scp:
+  - apiVersion: v2
+    appVersion: 0.1.2
+    created: "2023-04-20T16:05:20.865619-07:00"
+    dependencies:
+    - condition: abacus.enabled
+      name: abacus
+      repository: oci://ghcr.io/opentdf/charts
+      version: 1.2.0-sha83c261c
+    - name: attributes
+      repository: oci://ghcr.io/opentdf/charts
+      version: 0.0.0-sha-da7b457
+    - name: entitlement-pdp
+      repository: oci://ghcr.io/opentdf/charts
+      version: 0.0.0-sha-da7b457
+    - name: entitlement-store
+      repository: oci://ghcr.io/opentdf/charts
+      version: 0.0.0-sha-da7b457
+    - name: entitlements
+      repository: oci://ghcr.io/opentdf/charts
+      version: 0.0.0-sha-da7b457
+    - name: entity-resolution
+      repository: oci://ghcr.io/opentdf/charts
+      version: 0.0.0-sha-da7b457
+    - name: kas
+      repository: oci://ghcr.io/opentdf/charts
+      version: 0.0.0-sha-da7b457
+    - name: common-lib
+      repository: file://../common-lib
+      version: 0.1.0
+    - condition: embedded.postgresql
```

Comment:
embedded postgresql is not supported for production environments.

Reply:
If you want to use external postgres you can set embedded.postgresql to false